examples/imagenet/input_pipeline.py

```diff
@@ -216,9 +220,11 @@ def decode_example(example):
 
   if train:
     ds = ds.repeat()
-    ds = ds.shuffle(16 * batch_size, seed=0)
+    ds = ds.shuffle(16 * batch_size)
```

Comment:
Why get rid of the seed here?

Reply:
sorry my bad. I was trying to introduce the seeds using jax.random.split(seed) for reproducibility. Removed it assuming I introduced it. 

Thanks for catching it.